addons/website/static/src/scss/website.scss

```diff
@@ -1211,6 +1211,11 @@ header {
     border-radius: $rounded-pill;
     vertical-align: top;
 }
+body:not(.o_connected_user) {
+    .o_add_language {
+        display: none !important; // TODO the seperator sould not be in the DOM if not connected
+    }
+}
```

Comment:
Can't you say "if last child display none" ? It would be more correct as here a connected user which does not have the right for the add language button will still see that separator.

Also: "seperator"